### PR TITLE
fix: add index.ts to graphqlgen.yml

### PR DESCRIPTION
**Pull request by timsuchanek**

### CODE REVIEW

File: graphqlgen.yml [hunk 1]
```diff
@@ -9,14 +9,16 @@ context: ./src/types.ts:Context
 
 # Map SDL types from the GraphQL schema to TS models
 models:
+  files:
```
Comment (timsuchanek) [inline graphqlgen.yml:12]: This is the crucial part :)

File: prisma/prisma.yml [hunk 2]
```diff
@@ -1,25 +1,24 @@
...
-datamodel: 
+datamodel:
   - types.graphql
   - enums.graphql
 
 generate:
   - generator: typescript-client
     output: ../src/generated/prisma-client/
- 
+  - generator: graphql-schema
```
Comment (timsuchanek) [inline prisma/prisma.yml:11]: we also have a schema generator so you don't need graphql get-schema!

Comment (JensMadsen) [reply prisma/prisma.yml:11]: great :-) . I did not realise that. It is not in your tutorials yet as far as I can see

Comment (JensMadsen) [reply prisma/prisma.yml:11]: Also, when called from prisma deploy using prisma.yml it returns success (green check mark ) even though the command was not successful

Comment (JensMadsen) [reply prisma/prisma.yml:11]: ... sorry about all these comment. How do I run the prisma schema generator?

Comment (JensMadsen) [reply prisma/prisma.yml:11]: even more sorry. I guess it s the added generator part in prisma.yml :-)

**Pull request merged**